src/tests/meson.build

```diff
@@ -0,0 +1,75 @@
+# Included from top-level meson.build
+
+## Includes
+for_tests_incdir = include_directories(
+    'includes/',
+    '../../dasynq/include/'
+)
+#ToDo; we need to have another meson.build for cptests
```

Comment:
Is this message still accurate?

Reply:
There is a `meson.build` inside cptests so the comment doesn't seem to make sense

Reply:
`meson.build` in `cptests` dir is for fuzzer target! I'll send a PR about spliting fuzz target directory and cptests directory. Then I remove this message.

Reply:
hang on, why split fuzz target directory from cptests directory? They are both control protocol tests.
Meson should be able to handle this.

Reply:
Hmm; Creating a `fuzz` directory in `cptests` dir does not fix our issue? If your your opinion is negative; I'll search for an another way.

Reply:
Well, it shouldn't be necessary.  If meson can't handle having two targets in one directory then it's a pretty limited build system (this corresponds to my previous experience with meson, which is part of why I'm resistant to it in the first place).

I don't like either that the current situation has a completely separate build file in cptests which is just for fuzzing. That's wrong. If that can't be fixed in meson, I suggest removing the support for the fuzz target from the meson build altogether. You could do that as a first step anyway and if you can figure out how to get it to work later, raise a second PR then.

(I know the makefile build doesn't currently allow building the fuzzer from the top-level makefile, but it could trivially be made to do so).

Reply:
Meson should not care how many targets you have in a single directory...

Reply:
I think about it. Seems like my implementation about fuzzing in meson is not fit for our purposes. Also I suggest don't merge this PR until all known issues will fixed.

Reply:
I refactored `fuzz` and `cptests` and now they are living in a same directory. @davmac314 That's ok?

Reply:
Yes that looks ok, thanks. I have a few other comments still, I'll add separately.

Reply:
Ok; I close this